src/ui/components/LedgerViewer.js

```diff
@@ -48,6 +48,12 @@ const useStyles = makeStyles((theme) => {
       paddingLeft: theme.spacing(2),
       paddingRight: theme.spacing(1),
     },
+    chip: {
+      fontSize: "0.55rem",
+      color: "#828282",
+      backgroundColor: "#383838",
```

Comment:
Should use values from the theme instead of hardcoded ones

Reply:
These are one-off colors. Do they still belong in the theme? If so, where do they go?